api/config/featured_workspaces_prod.json

```diff
@@ -30,6 +30,11 @@
       "namespace": "aou-rw-edae682b",
       "id": "masterdepressionphenotype",
       "category": "PHENOTYPE_LIBRARY"
+    }, {
+      "name": "Versioning notebooks and intermediate results",
+      "namespace": "aou-rw-c73447b2",
+      "id": "featuredworkspaceversioningnotebooksandintermediateresults",
+      "category": "TUTORIAL_WORKSPACES"
```

Comment:
nit: might want to keep the categories together.

I'd rather have a randomly generated ID than a sentence with no spaces, but that's just me.

Reply:
sure. keeping it together sounds good to me. I don't have control over the ID unfortunately.